packages/types/src/index.ts

```diff
@@ -121,181 +137,170 @@ export type MultiSelect =
 
 export interface Action {
   type: string;
+  /**
+   * @description: An identifier for this action. You can use this when you receive an interaction payload to
+   * {@link https://api.slack.com/interactivity/handling#payloads identify the source of the action}. Should be unique
+   * among all other `action_id`s in the containing block. Maximum length for this field is 255 characters.
+   */
   action_id?: string;
 }
 
-export interface UsersSelect extends Action {
-  type: 'users_select';
-  initial_user?: string;
-  placeholder?: PlainTextElement;
+export interface Confirmable {
+  /**
+   * @description A {@see Confirm} object that defines an optional confirmation dialog after the element is interacted
+   * with.
+   */
   confirm?: Confirm;
+}
+
+export interface Focusable {
+  /**
+   * @description Indicates whether the element will be set to auto focus within the
+   * {@link https://api.slack.com/reference/surfaces/views `view` object}. Only one element can be set to `true`.
+   * Defaults to false.
+   */
   focus_on_load?: boolean;
 }
 
-export interface MultiUsersSelect extends Action {
+export interface Placeholderable {
```

Comment:
```suggestion
export interface Placeholdable {
```
Too many syllables for my little brain